src/main/java/utils/GetCredentials.java

```diff
@@ -0,0 +1,43 @@
+package utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class GetCredentials {
```

Comment:
It would be good not to name class starting from verb, as it is an entity to create an object